x/marketmap/types/market_test.go

```diff
@@ -383,9 +383,9 @@ func TestValidateMarketMap(t *testing.T) {
 			name: "second operation is not an index price provider",
 			cfg: types.MarketMap{
 				Tickers: map[string]types.Ticker{
-					constants.BITCOIN_USDT.String(): constants.BITCOIN_USDT,
-					constants.BITCOIN_USD.String():  constants.BITCOIN_USD,
-					constants.USDT_USD.String():     constants.USDT_USD,
+					constants.BITCOIN_USDT.String(): types.Ticker(constants.BITCOIN_USDT),
```

Comment:
Why do we need type casting in these?

Reply:
in this temp state, we have two types of Tickers (sadge)

marketmap.Ticker and mm2.Ticker

(this goes away once we move mm2 -> marketmap